code/lib/cli/src/automigrate/fixes/index.ts

```diff
@@ -48,6 +49,7 @@ export const allFixes: Fix[] = [
   reactDocgen,
   storyshotsMigration,
   removeReactDependency,
+  webpack5Migration,
```

Comment:
These are all migrations, so I don't think that the `xMigration` suffix is needed. How about `webpack5Compiler` since we are really migrating the compiler only.